contracts/external/interfaces/gmx/GmxWithdrawal.sol

```diff
@@ -0,0 +1,168 @@
+// SPDX-License-Identifier: BUSL-1.1
+
+pragma solidity ^0.8.9;
+
+/**
+ * @title Withdrawal
+ * @dev Struct for withdrawals
+ */
+library Withdrawal {
```

Comment:
Prefix the name with `Gmx`. 2 new-lines between contract/library name & pragma/imports